packages/spreadsheet-view/src/SpreadsheetView/components/SpreadsheetView.js

```diff
@@ -0,0 +1,221 @@
+export default pluginManager => {
+  const { jbrequire } = pluginManager
+  const { observer, PropTypes } = jbrequire('mobx-react')
+  const React = jbrequire('react')
+  const { useEffect, useState } = React
+  const { Icon, IconButton, InputAdornment } = jbrequire('@material-ui/core')
+  const { makeStyles } = jbrequire('@material-ui/core/styles')
+  const TextField = jbrequire('@material-ui/core/TextField')
+  const Grid = jbrequire('@material-ui/core/Grid')
+  const ResizeHandle = jbrequire('@gmod/jbrowse-core/components/ResizeHandle')
+
+  const ImportWizard = jbrequire(require('./ImportWizard'))
+  const Spreadsheet = jbrequire(require('./Spreadsheet'))
+
+  const headerHeight = 52
+  const statusBarHeight = 20
+
+  // stolen from https://dev.to/gabe_ragland/debouncing-with-react-hooks-jci
+  function useDebounce(value, delay) {
```

Comment:
we have a import { useDebounce } from '@gmod/jbrowse-core/util' that is identical

Reply:
oh nice, did not know that. fixed.